services/network-storage.rst

```diff
@@ -0,0 +1,104 @@
+Network Storage
+===============
+
+The network storage service provides a file store and enables a new kind of
+``mount`` that can be shared between different application containers.
+application containers.
+
+To use it in your application, add it to ``.symfony/services.yaml``:
+
+.. code-block:: yaml
+
+    files:
+        type: network-storage:1.0
+        disk: 256
+
+And wire it in ``.symfony.cloud.yaml``:
+
+.. code-block:: yaml
+
+    mounts:
+        '/var': { source: local, source_path: var }
+        '/public/uploads': { source: service, service: files, source_path: uploads }
+
+.. note::
+
+   You do **not** need to add a relationship to point to the ``files`` service.
+   This is handled automatically by SymfonyCloud.
+
+The application can now read from and write to the ``public/uploads`` directory
+as if it was a local writeable mount.
```

Comment:
I think we need to clearly state the difference between `var` and `/pub/uploads` in your example.

Reply:
updated. let me know if it is better now